src/main/java/dynamok/sink/DynamoDbSinkTask.java

```diff
@@ -94,6 +114,11 @@ public void put(Collection<SinkRecord> records) {
                     }
                 }
             }
+        } catch (LimitExceededException | ProvisionedThroughputExceededException e) {
+            log.debug("Batch write of {} records failed with Limit/Throughput Exceeded exception; backing off",
```

Comment:
And I appreciate your flexibility :)

Left this log.debug message without the extra printout of "e", since the framework will print it out as part of the handling of RetriableException.